src/js/toggle-btn-script.js

```diff
@@ -15,29 +15,33 @@
        */
       // eslint-disable-next-line one-var,no-var
       $('.toggle-list', context).once('toggleBtn').each(function () {
-        var $relatedArtList = $(this).children('ul'),
-          maxRel = parseInt($(this).attr('data-toggle-length'), 10 || 7),
-          relTotal = $relatedArtList.children('li').length,
-          available = relTotal - maxRel,
-          relToggleText = Drupal.t('Show @num more', {'@num': available});
+        let maxItems = parseInt($(this).attr('data-toggle-length'), 10);
+        if (isNaN(maxItems)) maxItems = 7;
 
-        if (relTotal > maxRel) {
-          $relatedArtList.attr({ id: 'toggle-menu', 'aria-live': 'polite' });
+        let $toggleList = $(this).find('ul'),
```

Comment:
What's the convention with JavaScript variables vs PHP variables ($ prefix)? Or does the $ here hint of a jQuery object?

Reply:
Just me thinking out loud here by the way!

Reply:
Yep - there is a convention that variables containing jquery objects have $ prefix

Reply:
https://stackoverflow.com/questions/6209462/when-why-to-prefix-variables-with-when-using-jquery